test/fee/MintFeeManager.t.sol

```diff
@@ -14,13 +15,15 @@ import {SimpleMinter} from "../mock/SimpleMinter.sol";
 contract MintFeeManagerTest is Test {
```

Comment:
is `MintFeeManager even used anymore?  Isn't all the logic now in `ProtocolRewards` and if so can we delete the `MintFeeManager` class?